lib/kits/core-ui/components/paged-publication/main-container.js

```diff
@@ -0,0 +1,174 @@
+                        </path>
+                    </svg>
+                </button>
+                <button class="sgn__offer-shopping">
+                    <svg
+                        aria-hidden="true"
+                        class="sgn-header-icon-svg"
+                        role="img"
+                        viewBox="0 0 512 512"
+                        xmlns="http://www.w3.org/2000/svg"
+                    >
+                        <path
+                            fill="currentColor"
+                            d="M48 368a48 48 0 1 0 48 48 48 48 0 0 0-48-48zm0-160a48 48 0 1 0 48 48 48 48 0 0 0-48-48zm0-160a48 48 0 1 0 48 48 48 48 0 0 0-48-48zm448 24H176a16 16 0 0 0-16 16v16a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16V88a16 16 0 0 0-16-16zm0 160H176a16 16 0 0 0-16 16v16a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-16a16 16 0 0 0-16-16zm0 160H176a16 16 0 0 0-16 16v16a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-16a16 16 0 0 0-16-16z"
+                        >
+                        </path>
+                    </svg>
+                    <div class="sgn__offer-shopping-count"><div>
+                </button>
+                <button class="sgn__nav-menu-btn">
+                    <svg
+                        aria-hidden="true"
+                        class="sgn-header-icon-svg"
+                        role="img"
+                        viewBox="0 0 512 512"
+                        xmlns="http://www.w3.org/2000/svg"
+                    >
+                        <path
+                            fill="currentColor"
+                            d="M304 256c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48 48 21.5 48 48zm120-48c-26.5 0-48 21.5-48 48s21.5 48 48 48 48-21.5 48-48-21.5-48-48-48zm-336 0c-26.5 0-48 21.5-48 48s21.5 48 48 48 48-21.5 48-48-21.5-48-48-48z"
+                        >
+                        </path>
+                    </svg>
+                </button>
+            </div>
+        </div>
+        <div class="verso">
+            <div class="verso__scroller">
+                <div class="sgn-pp__pages"></div>
+            </div>
+        </div>
+        <div class="sgn-pp__progress">
+            <div class="sgn-pp-progress__bar"></div>
+        </div>
+        <div class="sgn-pp__progress-label"></div>
+        <a
+            class="sgn-pp__control"
+            href="#"
+            role="button"
+            data-direction="prev"
+        >
+            &lsaquo;
+        </a>
+        <a
+            class="sgn-pp__control"
+            href="#"
+            role="button"
+            data-direction="next"
+        >
+            &rsaquo;
+        </a>
+    </div>\
+`;
+
+const defaultShoppingCountTemplate = `\
+{{#shoppingCount}}
+    <div class="sgn__offer-shopping-counter">
+        <span>{{shoppingCount}}</span>
+    </div>
+{{/shoppingCount}}\
+`;
+
+const mainContainer = ({template, shoppingCounterTemplate, el, scriptEls}) => {
+    const _template = template?.innerHTML || defaultTemplate;
```

Comment:
Could you elaborate on why you use _ in front of variables?

Reply:
This is only for the distinction of local and global variables especially when I was converting it from classes to functions. We can just remove it if you want to :)

Reply:
I'm thinking: 1) We shouldn't have global variables (should be within SGN) 2) _ is not pretty so if we can avoid it it would be nice 😅

Reply:
Sure, I'll refactor it to remove those _ :D